mod_upload/controllers.py

```diff
@@ -325,6 +325,10 @@ def process_id(upload_id):
                             )
                             g.db.add(issue)
                             g.db.commit()
+                        else:
+                            flash('Could not submit an issue on GitHub '
```

Comment:
You add a flash message, but I see no changes to the templates to show the flashed message.